ocs_ci/deployment/vmware.py

```diff
@@ -752,11 +752,14 @@ def destroy_cluster(self, log_level="DEBUG"):
         # remove csi users in case of external deployment
         if config.DEPLOYMENT["external_mode"]:
             logger.debug("deleting csi users")
-            toolbox = pod.get_ceph_tools_pod()
-            toolbox.exec_cmd_on_pod("ceph auth del client.csi-cephfs-node")
-            toolbox.exec_cmd_on_pod("ceph auth del client.csi-cephfs-provisioner")
-            toolbox.exec_cmd_on_pod("ceph auth del client.csi-rbd-node")
-            toolbox.exec_cmd_on_pod("ceph auth del client.csi-rbd-provisioner")
+            try:
+                toolbox = pod.get_ceph_tools_pod()
+                toolbox.exec_cmd_on_pod("ceph auth del client.csi-cephfs-node")
+                toolbox.exec_cmd_on_pod("ceph auth del client.csi-cephfs-provisioner")
+                toolbox.exec_cmd_on_pod("ceph auth del client.csi-rbd-node")
+                toolbox.exec_cmd_on_pod("ceph auth del client.csi-rbd-provisioner")
+            except AssertionError as ex:
+                logger.debug(ex)
```

Comment:
What about to add some more log message before like:
logger.warning("Failed to setup the Ceph toolbox pod. Probably because the installation was not successful")

Reply:
done

Reply:
Wait, why do we even care to run these commands via ceph toolbox, when we ignore any errors which may happen during this process?